api/src/test/resources/org/openmrs/api/include/DiagnosisServiceImplTest-SetupDiagnosis.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<dataset>
+	<diagnosis diagnosis_id="1" patient_id="2" encounter_id="1" creator="1" date_created="2015-01-12 00:00:00" 
```

Comment:
@pluwum  I think this will throw. Are you sure the table name is `diagnosis`?
Also, the rank is required.

Reply:
@paradisekelechi fixed that. 🙇